tox.ini

```diff
@@ -3,12 +3,13 @@ envlist = py3{5,6,7,8,9}
 
 [testenv]
 deps =
-  mypy
+  mypy @ git+https://github.com/python/mypy@538d36481526135c44b90383663eaa177cfc32e3
```

Comment:
We should probably make an issue to track switching back to a stable release when we can. It looks like this commit will be in v0.800

Reply:
https://github.com/stlehmann/PyQt5-stubs/issues/120